inc/integrations/class-pico.php

```diff
@@ -144,14 +144,14 @@ public function render_pico_auth_key_input() {
 	}
 
 	/**
-	 * Render an input for the Pico Whitelisted SSO Tiers.
+	 * Render a textarea for the Pico allow listed SSO Tiers.
 	 */
 	public function render_pico_tiers_input() {
 		// Check to see if there are existing tiers in the option.
 		$tiers = $this->options[ $this->option_key ]['tiers'] ?? '';
 
 		?>
-			<input id="pico_tiers" type="text" name="irving_integrations[<?php echo esc_attr( 'pico_tiers' ); ?>]" value="<?php echo esc_attr( $tiers ); ?>" />
+			<textarea id="pico_tiers" rows="5" cols="50" type="text" name="irving_integrations[<?php echo esc_attr( 'pico_tiers' ); ?>]"><?php echo esc_attr( $tiers ); ?></textarea>
```

Comment:
Thank you. One other quick thing, could we trim whitespace + any trailing commas on this input during save or fetch? I want to handle all these situations,

- `'one,two,three' => [ 'one', 'two', 'three' ]` 👍 
- `'one, two, three' => [ 'one ', 'two ', 'three ' ]` 👎 
- `'one, two, three, ' => [ 'one', 'two', 'three', '']` 👎 
- `'one, two, three,' => [ 'one', 'two', 'three', '']` 👎 

I think an explode( ',', $values) and array_values() should just about handle it?

Could also be a new ticket if we prefer.

Reply:
Ah yes, thanks for the reminder, this has bugged me in the past too. Fixing